tests/cxx/funcptrs.cc

```diff
@@ -79,6 +80,14 @@ void FreeFunctions() {
   // IWYU: Retval is...*funcptrs-i1.h
   // IWYU: FunctionTemplate is...*funcptrs-i1.h
   &FunctionTemplate<Retval>;
+
+  // No need for full return type info for non-templated function...
+  // IWYU: FunctionReturningRecordType is...*funcptrs-i1.h
+  &FunctionReturningRecordType;
+  // ... and for function template specialization references.
```

Comment:
"or" instead of "and"?

Reply:
"Neither ... nor ..." is more clear to me.

Reply:
"Neither ... nand ..." maybe :)